samples/testbase/manage_package.py

```diff
@@ -0,0 +1,119 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import time
+
+# Use `from azure.identity import AzureCliCredential` if authenticating via Azure CLI for test
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import BlobClient
+
+from azure.mgmt.testbase import TestBase
+from azure.mgmt.testbase.models import PackageResource
+from azure.mgmt.testbase.models import TargetOSInfo
+from azure.mgmt.testbase.models import Test
+from azure.mgmt.testbase.models import Command
+from azure.mgmt.testbase.models import GetFileUploadURLParameters
+from testbase_util import format_json
+
+
+def main():
+    # Requesting token from Azure
+    print("Requesting token from Azure...")
+    # For other authentication approaches, please see: https://pypi.org/project/azure-identity/
+    #     Use `credential = AzureCliCredential()` if authenticating via Azure CLI for test
+    credential = DefaultAzureCredential()
+
+    # Run `export SUBSCRIPTION_ID="<subscription-id>"` on Linux-based OS
+    # Run `set SUBSCRIPTION_ID=<subscription-id>` on Windows
+    subscription_id = os.environ.get("SUBSCRIPTION_ID", None)
+
+    # Set variables
+    resource_group = "<resource-group-name>" # replace with real resource group name
+    testbase_account_name = "<testbaseaccount-name>" # replace with real test base account name
+    package_file_path = "<path-of-package-zip-file>" # replace with real path of the package zip file
+
+    # Create client
+    testbase_client = TestBase(credential, subscription_id)
+
+    # Create Package
+    print("Creating Package...")
+    timestamp = time.strftime("%m%d%H%M", time.localtime())
+    application_name = "contoso_package"
+    version = timestamp
+    package_name = "{name}-{version}".format(name=application_name, version = version)
+
+    target_os_info_su = _get_target_os_info("Security updates")
+    target_os_info_fu = _get_target_os_info("Feature updates")
```

Comment:
If this samlple can start from creating resource group and testbase account, it will be more perfect.

Reply:
> If this samlple can start from creating resource group and testbase account, it will be more perfect.

we have a separate sample for test base account creation @BigCat20196